department-of-veterans-affairs/caseflow
# APPEALS-29401 - refactor to fix console warnings in QueueTable

Resolves [APPEALS-29401](https://jira.devops.va.gov/browse/APPEALS-29401)

# Description
Deletes a console warning that seems pointless
Moves a call to requestTasks in QueueTable from the constructor to componentDidMount to avoid errors about setState being called from an unmounted component

## Acceptance Criteria
- [ ] Two fewer console warnings: the setState mentioned in the story, and the LoadingIcon warning

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2.  Make sure the whole queue didn't blow up. Log in as CAMOADMIN and check the queue to make sure cases load



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
